script/generate_build_info.py

```diff
@@ -130,6 +131,28 @@ def get_maintainer_from_dockerfile(dir_name):
                     return maintainer
     return "Unknown"
 
+def get_default_build_script(build_scripts_versions):
+    #print(build_scripts_versions[-1]['version'])
+    print((build_scripts_versions))
+    #def_script=''
+    result=[]
+    for data in build_scripts_versions:
+        if 'ubi_8.7' in data['file']:
+            #def_script=data['file']
+
+            #print(data['file'])
+            return [data['version'],data['file']]
+        else:
+            #def_script=data['file']
+            result.append(data['version'])
+            result.append(data['file'])
+    return result
+
+
+
+#print(type(data))
```

Comment:
The logic of function `get_default_build_script` looks ambiguous. Can you please revisit and check?